src/index.js

```diff
@@ -126,6 +128,28 @@ Object.assign(
 )
 
 /*
+ * This creates a glamorousComponentFactory for every jsxstyle element so you
+ * can simply do:
+ * const GreenBlock = glamorous.block({
+ *   backgroundColor: 'green',
+ *   padding: 20,
+ * })
+ * <GreenBlock>Click Me!</GreenBlock>
+ */
+Object.assign(
+  glamorous,
+  jsxStyleComponentTags.reduce(
+    (getters, tag) => {
+      const capitalTag = capitalize(tag)
```

Comment:
The other components are capitalized, so I followed suit. For that reason I say capitalize, but I don't have a strong opinion either way.